runtimes/runtimes/standalone.ts

```diff
@@ -246,6 +247,9 @@ export const standalone = (props: RuntimeProps) => {
                 onHover: handler => lspConnection.onHover(handler),
                 onSignatureHelp: handler => lspConnection.onSignatureHelp(handler),
                 extensions: {
+                    onGetAvailableServerConfigurations(handler) {
```

Comment:
I'd suggest to remove `Server` from `getAvailableServerConfigurations` naming. When I read `ServerConfiguration`, I think of it as something that allows me to configure server, which is not the intent here.